### DISCUSSION TITLE
some question about hydrogen

### DISCUSSION THREAD

**Discussion by aixi-aizi** [Slimefun-Addon-Community/Galactifun#119] [General]
Recently, I found that this module has some combustible gases, so I obtained some hydrogen by electrolyzing water, but I found that the core of the launch pad does not send hydrogen into the rocket, but oil can. Is it that ordinary rockets can not use hydrogen as fuel, or I encounter a bug? I wonder if you could help me with this.

**Comment by Seggan** [comment]
What version of Galactifun are you using?

**Reply by aixi-aizi** [reply]
well,Im just a player.I will ask the server administrators about this tomorrow.

**Reply by aixi-aizi** [reply]
Is V32 the true version of Galactifun?
![image](https://user-images.githubusercontent.com/98971365/153543906-58c44e91-4baa-40a5-8785-7b6724d50663.png)

**Reply by Seggan** [reply]
Thats a very old version

**Reply by aixi-aizi** [reply]
ok，I have told the administrator about it ,he said will update Galactifun to the new version.Look forward to your future updates.
